react/features/rtcstats/RTCStats.ts

```diff
@@ -185,6 +198,38 @@ class RTCStats {
         this.trace?.close();
     }
 
+    /**
+     * RTCStats client can notify the APP of any PeerConnection related event that occurs.
+     *
+     * @param {Object} event - Rtcstats event.
+     * @returns {void}
+     */
+    handleRtcstatsEvent(event: any) {
+        switch (event.type) {
+        case PC_CON_STATE_CHANGE: {
+            const { body: { isP2P, state } = { state: null,
```

Comment:
Do we get any other data that would be worthy of extraction as part of the event?

Reply:
there's also a connection identifier (pcId) but didn't see any use for it for now.